source/_integrations/emulated_hue.markdown

```diff
@@ -156,6 +156,13 @@ You can verify that the `emulated_hue` integration has been loaded and is respon
 
 Verify that the URLs above are using port 80, rather than port 8300 (i.e., `http://<HA IP Address>:80/description.xml`). Both Google Home and Amazon Alexa/Echo (as of the 2019-08 firmware) require port 80.
 
+In builds 0.115 and forward `input_boolean` entities may no longer be discovered respond as expected by Alexa. To work around this issue add 
+
+```yaml
+  lights_all_dimmable: true
+```
+to the `emulated_hue` configuration.
```

Comment:
We don't specify versions in our documentation in general. The documentation should be up to par with the latest.

However, isn't this a bug?

Reply:
Thanks for the feedback. I would say so (it is a bug). where however would we post a workaround? This si a simple (and compatible) trick that seems to resolve the issue for now.  Happy to remove the version number of course if this si all it takes. Also happy to file this in a different way - just let me know.

Reply:
We don't document bugs, we report and fix them instead.

Therefore, this should not be added to the documentation.